include/ftsh_functions.h

```diff
@@ -0,0 +1,32 @@
+#ifndef FTSH_FUNCTIONS
+#define FTSH_FUNCTIONS
+
+/***************************************************************************//**
+
+	@file         ftsh_functions.h
+
+	@author       Max Bronckers
+
+	@date         Thursday, 1 May 2018
+
+	@brief        FTSH supported commands header file
+
+*******************************************************************************/
+
+#include <sys/wait.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
```

Comment:
I'd prefer header comments to either only be in the header file or be duplicated between the header and C files if necessary.